logstash-output-pagerduty.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-pagerduty'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
I don't think its worth a `1.1.0` release adding a test suite, it more like a `1.0.1`